.idea/compiler.xml

```diff
@@ -48,17 +48,22 @@
       <module name="fdb-extensions_integrationTest" target="1.8" />
       <module name="fdb-extensions_main" target="1.8" />
       <module name="fdb-extensions_test" target="1.8" />
-      <module name="fdb-record-layer-icu_integrationTest" target="1.8" />
-      <module name="fdb-record-layer-icu_main" target="1.8" />
-      <module name="fdb-record-layer-icu_test" target="1.8" />
       <module name="fdb-record-layer-core-shaded_integrationTest" target="1.8" />
       <module name="fdb-record-layer-core-shaded_main" target="1.8" />
       <module name="fdb-record-layer-core-shaded_test" target="1.8" />
       <module name="fdb-record-layer-core_integrationTest" target="1.8" />
       <module name="fdb-record-layer-core_main" target="1.8" />
       <module name="fdb-record-layer-core_test" target="1.8" />
+      <module name="fdb-record-layer-geophile_main" target="1.8" />
+      <module name="fdb-record-layer-geophile_test" target="1.8" />
```

Comment:
These two probably need to be removed after the rename

Reply:
Removed.